src/client.c

```diff
@@ -56,31 +56,31 @@ int clientSendHandshake(struct client *c)
 }
 
 /* Write out a request. */
-#define REQUEST(LOWER, UPPER)                                       \
-	{                                                           \
-		struct message message;                             \
-		size_t n;                                           \
-		size_t n1;                                          \
-		size_t n2;                                          \
-		void *cursor;                                       \
-		ssize_t rv;                                         \
-		n1 = message__sizeof(&message);                     \
-		n2 = request_##LOWER##__sizeof(&request);           \
-		n = n1 + n2;                                        \
-		buffer__reset(&c->write);                           \
-		cursor = buffer__advance(&c->write, n);             \
-		if (cursor == NULL) {                               \
-			return DQLITE_NOMEM;                        \
-		}                                                   \
-		assert(n2 % 8 == 0);                                \
-		message.type = DQLITE_REQUEST_##UPPER;              \
-		message.words = (uint32_t)(n2 / 8);                 \
-		message__encode(&message, &cursor);                 \
-		request_##LOWER##__encode(&request, &cursor);       \
-		rv = write(c->fd, buffer__cursor(&c->write, 0), n); \
-		if (rv != (int)n) {                                 \
-			return DQLITE_ERROR;                        \
-		}                                                   \
+#define REQUEST(LOWER, UPPER)                                     \
+	{                                                         \
+		struct message message;                           \
+		size_t n;                                         \
+		size_t n1;                                        \
+		size_t n2;                                        \
+		void *cursor;                                     \
+		ssize_t rv;                                       \
+		n1 = messageSizeof(&message);                     \
+		n2 = request_##LOWER##Sizeof(&request);           \
```

Comment:
This is still mixed between snake_case and camelCase, it should be something like `request##CAPITAL##Sizeof(&request)`, where `CAPITAL` is the capital version of the message type name, replacing the `LOWER` version.